tests/visual-tests/playground/Clipboard.visual.test.tsx

```diff
@@ -0,0 +1,225 @@
+import dd from 'ts-dedent';
+
+import {expect, test} from 'playwright/core';
+
+import {Playground} from './Playground.helpers';
+
+test.describe('Clipboard', () => {
+    test.beforeEach(async ({mount, editor}) => {
+        await mount(<Playground />);
+        await editor.clearContent();
+    });
+
+    const emphasisMarkup = dd`
+    ## Emphasis
+
+    **This is bold text**
+
+    *This is italic text*
+
+    ~~Strikethrough~~
+    `;
+
+    test('should copy from wysiwyg and paste formatted to markup mode', async ({
```

Comment:
check screens plz

Reply:
https://github.com/gravity-ui/markdown-editor/pull/689/files#diff-a21b1aea17673947535aeb02733955782176b155bbddc61291d9e8a2734c5c7e is empty